roles/testnode/tasks/apt/repos.yml

```diff
@@ -1,4 +1,17 @@
 ---
+# Check for and remove custom repos.
+# http://tracker.ceph.com/issues/12794
+- name: Check for custom repos
+  shell: "ls -1 /etc/apt/sources.list.d/"
+  register: custom_repos
+  changed_when: false
+
+- name: Remove custom repos
```

Comment:
Does this need a ``when``?  What happens if there are no repos in /etc/apt/sources.list.d?